src/Sentry/Scope.cs

```diff
@@ -245,6 +245,22 @@ public void AddBreadcrumb(Breadcrumb breadcrumb)
         /// </summary>
         public void AddAttachment(Attachment attachment) => _attachments.Add(attachment);
 
+
+        /// <summary>
+        /// Clear all Attachments.
+        /// </summary>
+        public void ClearAttachments()
+        {
+#if NETCOREAPP3_0_OR_GREATER || NETSTANDARD2_1 || NET5_0_OR_GREATER
```

Comment:
Doesn't `NETCOREAPP3_0_OR_GREATER` include `NET5_0_OR_GREATER` ?


```suggestion
#if NETCOREAPP3_0_OR_GREATER || NETSTANDARD2_1
```

Ideally, this would be inverted like: `#if NETSTANDARD2_0 || NET461`.
The reasoning is: We special case to the platforms that don't support the best solution. Anything else takes the ideal solution. If we add a new target in the future it'll automatically adopt the "else" part which is what we want.